x-pack/plugins/data_usage/server/services/autoops_api.ts

```diff
@@ -52,12 +52,23 @@ export class AutoOpsAPIService {
       throw new AutoOpsError(AUTO_OPS_MISSING_CONFIG_ERROR);
     }
 
+    if (!autoopsConfig.api?.url) {
```

Comment:
Updated in response to PR review comment https://github.com/elastic/kibana/pull/200126/files#r1846228792